staging/src/k8s.io/apiserver/pkg/server/healthz/healthz.go

```diff
@@ -252,8 +252,11 @@ func handleRootHealth(name string, firstTimeHealthy func(), checks ...HealthChec
 			}
 		}
 		if excluded.Len() > 0 {
-			fmt.Fprintf(&individualCheckOutput, "warn: some health checks cannot be excluded: no matches for %s\n", formatQuoted(excluded.List()...))
-			klog.Warningf("cannot exclude some health checks, no health checks are installed matching %s",
+			verbosity := klog.V(6)
+			if verbosity.Enabled() {
+				fmt.Fprintf(&individualCheckOutput, "warn: some health checks cannot be excluded: no matches for %s\n", formatQuoted(excluded.List()...))
+			}
+			verbosity.Infof("cannot exclude some health checks, no health checks are installed matching %s",
```

Comment:
I can see excluding this from the log, but I think we always want to return the warning in the response to the caller, right?

Reply:
good point, i will adjust. thanks!